Disable controllers testcases

The cpuset controllers testcases do not build due to bug
https://sourceforge.net/tracker/?func=detail&aid=3126942&group_id=3382&atid=103382. Disabling
just the cpuset controllers do not seem to be easily possible, and
those controller features are rarely used on embedded systems anyway.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 testcases/kernel/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ltp-testsuite-20101031/testcases/kernel/Makefile
===================================================================
--- ltp-testsuite-20101031.orig/testcases/kernel/Makefile
+++ ltp-testsuite-20101031/testcases/kernel/Makefile
@@ -28,7 +28,7 @@
 #
 # KEEP THIS LIST ALPHABETIZED PLEASE!
 ifneq ($(UCLINUX),1)
-SUBDIRS		:= containers connectors controllers fs hotplug io ipc mem \
+SUBDIRS		:= containers connectors fs hotplug io ipc mem \
 		   numa performance_counters power_management pty sched \
 		   security syscalls timers tracing
 else
